TemplateApplication/Supporting Files/TemplateApplication.docc/Setup.md

```diff
@@ -45,65 +39,68 @@ You can learn more about Xcode, including [creating an Xcode project for an app]
 
 @Image(source: "Scheme", alt: "Screenshot showing the application scheme Run configuration's launch arguments.")
 
-The login and account setup is skipped in this configuration.
 
+### Alternative B: Set Up the Firebase Emulator Suite
 
-### Alternative B: Firebase Setup
+The application also provides a [Firebase Firestore](https://firebase.google.com/docs/firestore)-based data upload mechanism and [Firebase Authentication](https://firebase.google.com/docs/auth) login & sign-up. If you wish to test this functionality, you will need to have the [Firebase Emulator Suite](https://firebase.google.com/docs/emulator-suite) installed and running. This tool emulates a cloud-based backend on your Mac and does not require that you have a Firebase account to use.
 
-The application also provides a [Firebase Firestore](https://firebase.google.com/docs/firestore)-based data upload and [Firebase Authentication](https://firebase.google.com/docs/auth) login & sign-up.
-It is required to have the [Firebase Emulator Suite](https://firebase.google.com/docs/emulator-suite) to be up and running to use these features to build and test the application locally.
 The setup script described above installs the [Firebase Emulator Suite](https://firebase.google.com/docs/emulator-suite).
 
 > Important: You do not have to make any modifications to the Firebase configuration, log into the `Firebase` CLI using your Google account, or create a project in Firebase to run, build, and test the application!
 
-Navigate to the root folder of this setup containing your **.xcodeproj** file ([using `cd` in your terminal](https://tutorials.codebar.io/command-line/introduction/tutorial.html)) and startup the [Firebase Emulator Suite](https://firebase.google.com/docs/emulator-suite) in your [Terminal](https://support.apple.com/guide/terminal/open-or-quit-terminal-apd5265185d-f365-44cb-8b09-71a064a42125/mac) using
+Navigate to the root folder of this setup containing your **.xcodeproj** file ([using `cd` in your terminal](https://tutorials.codebar.io/command-line/introduction/tutorial.html)) and start the [Firebase Emulator Suite](https://firebase.google.com/docs/emulator-suite) in your [Terminal](https://support.apple.com/guide/terminal/open-or-quit-terminal-apd5265185d-f365-44cb-8b09-71a064a42125/mac) using
 ```bash
 $ firebase emulators:start
 ```
 
 @Image(source: "FirebaseCLI", alt: "Screenshot showing the terminal and the running Firebase Emulators.")
 
-After the emulators have started up, you can run the application in your simulator to build, test, and run the application and see the results show up in Firebase.
+After the emulators have started up, you can open the web interface by navigating to `http://127.0.0.1:4000/` in your web browser. When you run the Spezi Template Application in the next step, you will be able to use the application and see data populating in the emulator.
```

Comment:
Might be cool to make this a link if `DocC` renders that correctly or just remove the backticks.
```suggestion
After the emulators have started up, you can open the web interface by navigating to [`http://127.0.0.1:4000/`](http://127.0.0.1:4000/) in your web browser. When you run the Spezi Template Application in the next step, you will be able to use the application and see data populating in the emulator.
```

Reply:
The only downside is that it'll break the markdown link checker, but not a huge deal.

Reply:
I tried this but realized it breaks the markdown link check, which is a required check, so will revert it in order to merge.